force-app/main/default/objects/Test_Definition__mdt/fields/Academic_Level__c.field-meta.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Academic_Level__c</fullName>
+    <description>The grade-level, developmental level, or age-level that the Test Definition was designed for.</description>
+    <externalId>false</externalId>
+    <fieldManageability>DeveloperControlled</fieldManageability>
```

Comment:
Let's open up the fields on all of these to allow for any user to modify them. Same for test score.

Reply:
just confirming that we are ok with not push upgradable?

Reply:
These are represented as data and would have potentially been custom objects in a different world. So yes.